extensions/ext/GL_EXT_maximal_reconvergence.txt

```diff
@@ -0,0 +1,109 @@
+Name
+
+  EXT_maximal_reconvergence
+
+Name Strings
+
+  GL_EXT_maximal_reconvergence
+
+Contact
+
+  Alan Baker, Google LLC (alanbaker 'at' google.com)
+
+Contributors
+
+  Alan Baker, Google LLC
+
+Status
+
+  Draft.
+
+Version
+
+  Last Modified Date:   December 14, 2023
+  Revision: 1
+
+Number
+
+  TBD.
+
+Dependencies
+
+  This extension can be applied to OpenGL GLSL versions 1.40
+  (#version 140) and higher.
+
+  This extension can be applied to OpenGL ES ESSL versions 3.10
+  (#version 310) and higher.
+
+  This extension is written against the OpenGL Shading Language
+  Specification, version 4.60 (revision 7), dated July 10, 2019.
+
+  Interacts with GL_KHR_vulkan_glsl.
+
+  Interacts with GL_EXT_control_flow_attributes.
+
+Overview
+
+  This extension adds the ability to annotate entry points via an attribute to
+  indicate that subgroups should reconverge in the same way as invocation
+  groups.
+
+  This is done through the attribute syntax "[[ ... ]]" syntax. For example,
+    
+    void main() [[maximally_reconverges]] { ...  }
```

Comment:
I just noticed I spelled this differently in the glslang code I pushed this morning, `[[maximal_reconvergence]]`. I think I may have written that code before the actual spec was drafted. Should I change glslang or do you want to change the spec?

Reply:
I think I used `[[maximally_reconverges]]` in the blog post so a slight preference to update glslang, but we could also update the blog.

Reply:
OK, I'll fix glslang.

Reply:
This fix was merged: https://github.com/KhronosGroup/glslang/pull/3489